Library/Homebrew/dev-cmd/generate-cask-api.rb

```diff
@@ -0,0 +1,62 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cli/parser"
+require "cask/cask"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  sig { returns(CLI::Parser) }
+  def generate_cask_api_args
+    Homebrew::CLI::Parser.new do
+      description <<~EOS
+        Generates Cask API data files for formulae.brew.sh.
```

Comment:
```suggestion
        Generates Cask API data files for formulae.brew.sh.
        
        The generated files are written to the current directory.
```

Thoughts on including this? Could even go further and clarify which directories, but probably not necessary